config/management/genesis/src/storage_helper.rs

```diff
@@ -198,7 +201,11 @@ impl StorageHelper {
     }
 
     #[cfg(test)]
-    pub fn set_layout(&self, path: &str, namespace: &str) -> Result<crate::layout::Layout, Error> {
+    pub fn set_layout(
+        &self,
+        path: &str,
+        namespace: &str,
+    ) -> Result<libra_management::layout::Layout, Error> {
```

Comment:
nit: if we're going to just import libra_management things directly (e.g., libra_management::secure_backend::DISK), we could just import layout too :)

Reply:
Yeah, my IDE didn't like it before but now it does 🤷🏻‍♂️